include/tvm/relax/expr.h

```diff
@@ -224,7 +224,7 @@ class TupleNode : public ExprNode {
     if (fields.size() == other->fields.size() && fields.size() == 0) {
       return true;
     } else {
-      equal->MarkGraphNode();
+      // equal->MarkGraphNode();
```

Comment:
Why do you comment these three lines?

Reply:
This is a temporary method proposed by @Ruihang Lai. 
Now assert_structural_equal reports error on two equivalent Tuple of ShapeExpr because it uses pointer equality checks, and that makes our structure check about tuples not pass. Like this script
```
from tvm.script.parser import ir as I, relax as R, tir as T
from tvm.ir.base import assert_structural_equal

@I.ir_module
class B1:
    @R.function
    def main(x: R.Tuple(R.Tensor((3, 3), "float32"), R.Tensor((3, 3), "float32"))):
        with R.dataflow():
            lv1 = x[0]
            R.output(lv1)
        return lv1

@I.ir_module
class B2:
    @R.function
    def main(x: R.Tuple(R.Tensor((3, 3), "float32"), R.Tensor((3, 3), "float32"))):
        with R.dataflow():
            lv1 = x[0]
            R.output(lv1)
        return lv1

x1 = B1["main"].params[0]
x2 = B2["main"].params[0]

assert_structural_equal([x1.shape, x1.shape], [x1.shape, x2.shape], True)
```